opentelemetry-userevents-exporter/src/logs/realtime_logprocessor.rs

```diff
@@ -0,0 +1,45 @@
+#![allow(unused_imports, unused_mut, unused_variables, unused_must_use, dead_code)]
+
+use std::{sync::{Arc, Weak}, borrow::Cow};
+use std::fmt::Debug;
+
+use opentelemetry_api::{InstrumentationLibrary, logs::LogRecord, logs::LogResult};
+use opentelemetry_sdk::{export::logs::LogExporter, export::logs::LogData, export::logs::ExportResult};
+
+use crate::logs::exporter::*;
+use crate::logs::exporter::ExporterConfig;
+
+use std::cell::Cell; 
+
+#[derive(Debug)]
+pub struct RealTimeLogProcessor {
```

Comment:
maybe a better name ? I can think of `ReentrantExportProcessor` or `SynchronousExportProcessor`.
Also, given this is a public processor, lets document the need for this (or in other words, add a comment why the simple/batch built-ins are not sufficient).

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/Internal/ReentrantExportProcessor.cs#L23-L28 some wordings could be copied from here.

Reply:
Good point. Renamed to ReentrantExportProcessor, and added relevant comments.